### PR TITLE
[FIX] l10n_ke_edi_tremol: Make KE specific fields visible only in KE companies

### PR DESCRIPTION
**Pull request by moerradi**
Ensures that fields specific to Kenya are only displayed when a Kenyan company is selected.

Task-3525517

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_ke_edi_tremol/models/product.py [hunk 1]
```diff
@@ -14,6 +14,12 @@ class ProductTemplate(models.Model):
         string='KRA Item Description',
         help='Product code description needed when not 16% VAT rated. ',
     )
+    current_country_code = fields.Char(compute='_compute_current_country_code')
```
Comment (jco-odoo) [inline addons/l10n_ke_edi_tremol/models/product.py:17]: We already have a field fiscal_country_codes

Comment (moerradi) [reply addons/l10n_ke_edi_tremol/models/product.py:17]: it was added until 16.1 it's not available in 16.0

Comment (jco-odoo) [reply addons/l10n_ke_edi_tremol/models/product.py:17]: @moerradi Then let's only do it from saas-16.1

Comment (moerradi) [reply addons/l10n_ke_edi_tremol/models/product.py:17]: aright !

**Pull request closed without merging**